pkg/ovs/ovsconfig/default_windows.go

```diff
@@ -26,9 +26,13 @@ const (
 	defaultConnNetwork = "winpipe"
 	namedPipePrefix    = `\\.\pipe\`
 	// Wait up to 5 seconds when getting port.
-	defaultGetPortTimeout = 5 * time.Second
+	defaultGetPortTimeout    = 5 * time.Second
+	defaultOvsVersionMessage = "ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION"
 )
 
+func GetOvsVersionHelp() string {
+	return "ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION"
```

Comment:
`return defaultOvsVersionMessage` ?

Do you consider to make `defaultOvsVersionMessage` as public variable and use it in directly  `pkg/ovs/ovsconfig/ovs_client.go `

Reply:
figured the function would be easier long term since we might want dynamic information there

Reply:
if you feel strongly ill refactor to a variabl  , but otherwise i think better to use the func